service/src/main/java/org/mskcc/cmo/metadb/service/MetadbRequestService.java

```diff
@@ -19,6 +19,8 @@
     Boolean requestHasUpdates(MetadbRequest existingRequest, MetadbRequest request) throws Exception;
     Boolean requestHasMetadataUpdates(RequestMetadata existingRequestMetadata,
             RequestMetadata requestMetadata) throws Exception;
+    Boolean requestHasMetadataUpdates(String existingRequestMetadataJson,
```

Comment:
same as `Boolean requestHasMetadataUpdates(RequestMetadata existingRequestMetadata,
            RequestMetadata requestMetadata)` but takes different arguments

Reply:
[requested change] something I want to dig into a bit is whether we can still treat this incoming JSON as a `RequestMetadata` object and either add a method to the request data factory that parses it as an instance of `RequestMetadata`

Reply:
That's what this method is doing, parsing the json to `requestMetadata` and then calling `requestHasMetadataUpdates(RequestMetadata existingRequestMetadata, RequestMetadata requestMetadata)`